pkg/featuremanager/cell.go

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package featuremanager
+
+import (
+	"github.com/cilium/hive/cell"
+	"github.com/spf13/pflag"
+)
+
+// Cell allows to manage the FeatureManager.
+// A feature is a group of cell hooks that are grouped together and their
```

Comment:
I wonder if we could come up with a better name than "Feature" for these. even "DynamicFeature" would be better.